src/fio/FioEngine.ts

```diff
@@ -1331,7 +1331,11 @@ export class FioEngine extends CurrencyEngine<FioTools, SafeFioWalletInfo> {
       throw new Error('makeSpend Missing publicAddress')
     if (quantity == null) throw new NoAmountSpecifiedError()
 
-    const {
+    let { otherParams } = edgeSpendInfo
+    if (
+      otherParams == null ||
+      (otherParams != null && Object.keys(otherParams).length === 0)
```

Comment:
Optional fix.  Check for != null is not necessary as it’s assumed if the first condition of the || is false.